src/edit.js

```diff
@@ -40,12 +40,40 @@ import WikilocIframe from './iframe.js';
 export default function Edit( { attributes, setAttributes } ) {
 	return (
 		<div { ...useBlockProps() }>
+			<InspectorControls>
+				<PanelBody title={ __( 'Map settings', 'wikiblock' ) }>
+					
+					<TextControl label={ __( 'Map width', 'wikiblock' ) } onChange={ ( val ) => setAttributes( { mapWidth: parseInt(val) } ) } value={ attributes.mapWidth } type="number" />
```

Comment:
It's purely personal taste, but if an arrow function has only one argument you can skip the `( )`. I think it makes the code a bit easier to read, but other people may prefer to keep all functions looking the same.

```suggestion
					<TextControl label={ __( 'Map width', 'wikiblock' ) } onChange={ val => setAttributes( { mapWidth: parseInt(val) } ) } value={ attributes.mapWidth } type="number" />
```

Reply:
I like your approach, and I agree that removing the `( )` on arrow functions with only one parameter makes the code easier to read. Thanks!